restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java

```diff
@@ -475,6 +477,7 @@ public VoiceInterpreter(VoiceInterpreterParams params) {
         this.asImsUa = params.isAsImsUa();
         this.imsUaLogin = params.getImsUaLogin();
         this.imsUaPassword = params.getImsUaPassword();
+        this.timeout = params.getTimeout();
```

Comment:
this is maybe for later refactoring, why dont we save the ref to params directly, instead of replcaiting all params atts in VI?